### ISSUE TITLE
Hyper-parameters for training on MIMIC-CXR dataset

### ISSUE THREAD

**Issue by o0t1ng0o** [YIKUAN8/Transformers-VQA#12]
Hi @YIKUAN8,

May I ask for the hyper-parameters for training on MIMIC-CXR dataset, such as optimizer, learning rate and scheduler?

Thank you in advance.
